node/mock-test.ts

```diff
@@ -5,17 +5,22 @@ import os = require('os');
 import path = require('path');
 import cmdm = require('./taskcommand');
 import shelljs = require('shelljs');
-import syncRequest from 'sync-request';
+import deasync = require('deasync')
+const Downloader = require("nodejs-file-downloader");
```

Comment:
it's better to keep consistency between import formats

Reply:
Since nodejs-file-downloader has not default export class, it is not possible to use same import formats here

Reply:
[suggestion] then we probably use `import * as Downloader from 'nodejs-file-downloader';`
feel free to resolve conversation